src/functions.js

```diff
@@ -0,0 +1,95 @@
+import UpdateStorage from './Modules/updateStorage.js';
+import { addTask } from './Modules/addRemove.js';
+
+const displayTasks = (taskInject) => {
+  const tasks = JSON.parse(localStorage.getItem('storedStTask')) || [];
```

Comment:
it would be beneficial to refactor the code by creating a reusable function that handles retrieving tasks from local storage. This approach helps prevent repetition in lines [41](https://github.com/Mutalenic/to-do-list-review/pull/2/files#diff-99adff4a611723728cb0d72f3e02b658cd9ac5ac2446a56c37afa149648a318aR41), 49, 61, 73, and 86, making the code easier to maintain and promoting code reusability.